<h1 id="debugger"><a aria-hidden="true" class="anchor-heading" href="#debugger"><svg aria-hidden="true" viewBox="0 0 16 16"><use xlink:href="#svg-link"></use></svg></a>Debugger</h1>
<h2 id="debugging-outbound-network-issues"><a aria-hidden="true" class="anchor-heading" href="#debugging-outbound-network-issues"><svg aria-hidden="true" viewBox="0 0 16 16"><use xlink:href="#svg-link"></use></svg></a>Debugging outbound network issues</h2>
<ol>
<li>Sources tab, hit <code>Pause on exceptions</code> (stop sign with pause icon)</li>
<li>Reload the page, and code will pause on the first exception that happens</li>
<li>Hover over the failing line, and investigate
<ul>
<li>are args to the function call what you'd expect them to be? Does anything stand out? Why is an error being thrown?</li>
</ul>
</li>
</ol>